cloudinfo/catalog.go

```diff
@@ -516,3 +517,25 @@ func (infoSvc *CloudInfoService) GetOffering(catalogID string, offeringID string
 
 	return offering, response, err
 }
+
+func (infoSvc *CloudInfoService) GetOfferingRequiredInputs(offering *catalogmanagementv1.Offering, VersionID string, OfferingID string) (requiredInputs []string) {
```

Comment:
would this be more useful if it was GetOfferingInputs then return a list of input details? https://github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/blob/main/cloudinfo/projects_types.go#L181-L189